api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>May also be interesting to add functionality to test for whether the current value is anomalous for the month it is in? So this will detect if the current June's temperature is higher than all previous year's June's temperature. Due to the varying weather throughout the year, the current implementation may possibly only detect anomalies in months that tend to be colder, hotter, or wetter than other ones. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>yeah i have that in my code, but we decided to only include one test and show that you can add other tests... lets chat with justin if you think its important! im open to doing that.</p>